DESCRIPTION

```diff
@@ -28,6 +28,7 @@ Suggests:
     httpuv,
     knitr,
     plumber (>= 0.3.2),
+    quarto,
```

Comment:
This is a circular dependency, as `quarto` depends explicitly upon `rsconnect`.

Reply:
Before I added this, CI was failing because I use `quarto::quarto_path()` in tests. Should I just vendorize that function in the tests after all?

Reply:
Unfortunately, that feels best -- we don't care that the Quarto path comes from the `quarto` package, only that we can run the tests.